.github/workflows/build.yml

```diff
@@ -86,6 +86,67 @@ jobs:
           name: mpv-${{ matrix.target }}
           path: mpv-git-*.zip
 
+  win32:
+    runs-on: windows-latest
+    env:
+      VS: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise"
+      CC: "clang"
+      CXX: "clang++"
+      CC_LD: "lld"
+      CXX_LD: "lld"
+      WINDRES: "llvm-rc"
+    steps:
+      - name: Disable autocrlf
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
+
+      - uses: actions/checkout@v4
+
+      # https://github.com/mesonbuild/meson/pull/11715
+      - name: Install dependencies
+        run: |
+          git clone https://github.com/kasper93/meson --depth 1 -b 8981
```

Comment:
I've asked again, but I wouldn't hold my breath for it. The regression was introduced long time ago https://github.com/mesonbuild/meson/issues/8981 and my PR is also over a year old.

Reply:
Well, it does support it. depending if linker command line is converted to RSP or not. And meson limitations are not our fault. Either way I don't plan to advertise Windows SDK support anywhere. Like everything in mpv, things are there without any warranty.

Reply:
`compile-windows.md` needs big overhaul anyway. I don't touch compile guides, because every time you do it people will start asking questions and at some point you start explaining how to install windows... Compiling mpv with meson is trivial and there are no subordinary steps comparing to any other software.

But I hear what you are saying.